### PR TITLE
About page

### PR DESCRIPTION
**Pull request by Henrik-Bengtsson**

### CODE REVIEW

File: src/components/AboutComponent.vue [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+          <p class="text-center bg-[#E5E5E5] p-1 rounded-md mt-4">Fredrik</p>
+        </div>
+        <div class="flex flex-col">
+          <img class="h-32 w-32 rounded-full" src='../assets/about/head4.png' alt='blue'>
+          <p class="text-center bg-[#E5E5E5] p-1 rounded-md mt-4">Julia</p>
+        </div>
+        <div class="flex flex-col">
+          <img class="h-32 w-32 rounded-full" src='../assets/about/head5.png' alt='blue'>
+          <p class="text-center bg-[#E5E5E5] p-1 rounded-md mt-4">Henrik</p>
+        </div>
```
Comment (figgge) [inline src/components/AboutComponent.vue:38]:
```suggestion
      <div class="flex justify-evenly m-10 w-4/5">
        <div class="flex flex-col">
          <img class="h-32 w-32 rounded-full h-96 w-96" src='../assets/about/head1.png' alt='blue'>
          <p class="text-center bg-[#E5E5E5] p-1 rounded-md mt-4">Oliver</p>
        </div>
        <div class="flex flex-col">
          <img class="h-32 w-32 rounded-full h-96 w-96" src='../assets/about/head2.png' alt='blue'>
          <p class="text-center bg-[#E5E5E5] p-1 rounded-md mt-4">Robert</p>
        </div>
        <div class="flex flex-col">
          <img class="h-32 w-32 rounded-full h-96 w-96" src='../assets/about/head3.png' alt='blue'>
          <p class="text-center bg-[#E5E5E5] p-1 rounded-md mt-4">Fredrik</p>
        </div>
        <div class="flex flex-col">
          <img class="h-32 w-32 rounded-full h-96 w-96" src='../assets/about/head4.png' alt='blue'>
          <p class="text-center bg-[#E5E5E5] p-1 rounded-md mt-4">Julia</p>
        </div>
        <div class="flex flex-col">
          <img class="h-32 w-32 rounded-full h-96 w-96" src='../assets/about/head5.png' alt='blue'>
          <p class="text-center bg-[#E5E5E5] p-1 rounded-md mt-4">Henrik</p>
        </div>
```
The images are not the same size.

**Review by figgge — CHANGES_REQUESTED** [review]
Nice about page!

File: src/components/AboutComponent.vue (hunk 1 above)
Comment (Henrik-Bengtsson) [reply src/components/AboutComponent.vue:38]: Yes they are h-32 and w-32

Comment (Henrik-Bengtsson) [reply src/components/AboutComponent.vue:38]: and in my opinion 96 are to big

Comment (figgge) [reply src/components/AboutComponent.vue:38]: Seems like something wasn't updated on my end. Everything looks fine now... Great work!

**Review by figgge — APPROVED** [review]

**Pull request merged**